mne/report/report.py

```diff
@@ -2212,68 +2220,6 @@ def remove(self, *, title=None, tags=None, remove_all=False):
 
         return remove_idx
 
-    @fill_doc
-    def _add_or_replace(self, *, title, section, tags, html_partial, replace=False):
-        """Append HTML content report, or replace it if it already exists.
```

Comment:
why is this method being removed?  it's used a bunch of times later in the file. Revert.